ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -260,14 +260,21 @@ def get_workflow(dict_configs, cdump='gdas'):
 
     tasks = []
 
-    if app in ['S2S', 'S2SW']:
-        # Copy prototype ICs
-        deps = []
+    if app in ['S2S', 'S2SW', 'NG-GODAS']:
         base_cplic = dict_configs['coupled_ic']['BASE_CPLIC']
-
-        # ATM ICs
-        for file in ['gfs_ctrl.nc'] + [f'{datatype}_data.tile{tile_index}.nc' for datatype in ['gfs', 'sfc'] for tile_index in range(1, n_tiles + 1)]:
-            data = f"{base_cplic}/{dict_configs['coupled_ic'][f'CPL_ATMIC']}/@Y@m@d@H/&CDUMP;/{base.get('CASE','C384')}/INPUT/{file}"
+        if atm_type in ["MODEL"]:
+            deps = []
+            # ATM ICs
+            #base_cplic = dict_configs['coupled_ic']['BASE_CPLIC']
```

Comment:
Remove this now redundant line, and the other identical line below

Reply:
Done.